### PR TITLE
fix(ic): fixed the description of partInstanceId in case that there are several serial numbers available for a part

### PR DESCRIPTION
**Pull request by markuskeidl**
## Description

This PR fixes the documentation of the property partInstanceId in the Industry Core KIT. It clarifies which serial number should be used as partInstanceId for the part's digital twin in case that there are several serial numbers used by the manufacturer. 

It also removes 

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: docs-kits/kits/Industry Core Kit/Software Development View/page_digital-twins.mdx [hunk 1]
```diff
@@ -117,7 +117,7 @@ The following conventions for specific asset IDs apply to all digital twins:
...
-| partInstanceId | Mandatory    | The serial number of the part from the manufacturer.                                         | String |
+| partInstanceId | Mandatory    | The serial number of the part from the part's manufacturer. This must be the serial number that is applied to the part (as barcode or DMC) as the recipient (customer) of the part will use this information to lookup the part's digital twin in the manufacturer's DTR.<br/> A manufacturer might create more than one serial number for a part. For example, if there are customers that require the serial number to have a customer-specific format, the manufacturer might create two serial numbers for every part: an internal one that has a standard format for ease of use in the manufacturer's internal systems and a customer-specific one that is actually applied to the part. Only the customer-specific serial number is used as partInstanceId in this case. | String |
 | intrinsicId    | Mandatory    | This the unified technical key for searching a digital twin of a part. In this case the<br/> partInstanceId is used as the intrinsicId.                                         | String |
```
Comment (thomas-henn) [inline docs-kits/kits/Industry Core Kit/Software Development View/page_digital-twins.mdx:121]: What is meant by: "In this case the<br/> partInstanceId is used as the intrinsicId."
Which "case" is meant here? And in which cases is the intrinsicId to be used?

File: docs-kits/kits/Industry Core Kit/Software Development View/page_digital-twins.mdx [hunk 2]
```diff
@@ -117,7 +117,7 @@ The following conventions for specific asset IDs apply to all digital twins:
...
 | :------------- | :----------- | :------------------------------------------------------------------------------------------- | :----- |
-| partInstanceId | Mandatory    | The serial number of the part from the manufacturer.                                         | String |
+| partInstanceId | Mandatory    | The serial number of the part from the part's manufacturer. This must be the serial number that is applied to the part (as barcode or DMC) as the recipient (customer) of the part will use this information to lookup the part's digital twin in the manufacturer's DTR.<br/> A manufacturer might create more than one serial number for a part. For example, if there are customers that require the serial number to have a customer-specific format, the manufacturer might create two serial numbers for every part: an internal one that has a standard format for ease of use in the manufacturer's internal systems and a customer-specific one that is actually applied to the part. Only the customer-specific serial number is used as partInstanceId in this case. | String |
```
Comment (thomas-henn) [inline docs-kits/kits/Industry Core Kit/Software Development View/page_digital-twins.mdx:120]: Proposal:

The serial number of a part as specified by manufacturer of this part. This must be the serial number that is applied to the part, *e.g.*, as barcode or DMC etc., as the recipient, *e.g.*, customer of the part will use this information to lookup a digital twin of this part in the Digital Twin Registry of the manufacturer.<br/> A manufacturer might create more than one serial number for a part. For example, if there are customers that require the serial number to have a customer-specific format, the manufacturer might create two serial numbers for every part: an internal one that has a standard format for ease of use in the internal systems of a manufacturer, and a customer-specific one that is actually applied to the part. In this case, the customer-specific serial number is to be used as partInstanceId.

**Review by thomas-henn — COMMENTED** [review]
@markuskeidl : I propose refinements, and ask about the use of intrinsicId. This is unclear to the reader. Thank you.

File: docs-kits/kits/Industry Core Kit/Software Development View/page_digital-twins.mdx (hunk 2 above)
Comment (markuskeidl) [reply docs-kits/kits/Industry Core Kit/Software Development View/page_digital-twins.mdx]: Updated the PR

File: docs-kits/kits/Industry Core Kit/Software Development View/page_digital-twins.mdx [hunk 3]
```diff
@@ -117,7 +117,7 @@ The following conventions for specific asset IDs apply to all digital twins:
...
+| partInstanceId | Mandatory    | The serial number of a part as specified by the manufacturer of this part. This must be the serial number that is applied to the part, e.g., as barcode or DMC etc., as the recipient, e.g., customer, of the part will use this information to lookup a digital twin of this part in the Digital Twin Registry of the manufacturer.<br/>A manufacturer might create more than one serial number for a part. For example, if there are customers that require the serial number to have a customer-specific format, the manufacturer might create two serial numbers for every part: an internal one that has a standard format for ease of use in the internal systems of a manufacturer, and a customer-specific one that is actually applied to the part. In this case, the customer-specific serial number is to be used as partInstanceId. | String |
 | intrinsicId    | Mandatory    | This the unified technical key for searching a digital twin of a part. In this case the<br/> partInstanceId is used as the intrinsicId.                                         | String |
```
Comment (thomas-henn) [inline docs-kits/kits/Industry Core Kit/Software Development View/page_digital-twins.mdx:121]: Hi @markuskeidl , could you please modify this as a full sentence. I think there is at least one word missing.

**Review by thomas-henn — CHANGES_REQUESTED** [review]
Hi @markuskeidl , could you please specify the meaning of intrinsicId.

File: docs-kits/kits/Industry Core Kit/Software Development View/page_digital-twins.mdx (hunk 3 above)
Comment (markuskeidl) [reply docs-kits/kits/Industry Core Kit/Software Development View/page_digital-twins.mdx:121]: I just copied your sentence from above. Where is something missing? Can you point that out?

**Comment by eckardg** [comment]
I am not completely happy with that change. I would recommend to introduce a new specific asset id "customerPartInstanceId" that provides the serial number as described above. The old partInstanceId should still show the serial number used by the manufacturer.
For a new customerPartInstanceId we have to make sure that it can only be seen by that customer (similar to customerPartId).
In the case of a batch it could happen that one batch number would be delivered to several customers.

**Comment by stephanbcbauer** [comment]
@markuskeidl , @thomas-henn , @eckardg are these changes still needed?

**Comment by markuskeidl** [comment]
Yes, but I need to start working on it.

**Comment by eckardg** [comment]
Please also consider my comment

File: docs-kits/kits/Industry Core Kit/Software Development View/page_digital-twins.mdx (hunk 3 above)
Comment (markuskeidl) [reply docs-kits/kits/Industry Core Kit/Software Development View/page_digital-twins.mdx:121]: I reworked the sentences. I hope they are fine now.

**Comment by markuskeidl** [comment]
Gabi, your change would break the current standards and KITs. The lookup for serial parts is based on manufacturerId, manufacturerPartId/customerPartId, and partInstanceId. The lookup is done by the customer, so the customer only knows the data printed onto the part. That means that the serial number in partInstanceId must be the one that's printed on the part otherwise the lookup would not work.

So, partInstanceId should stay as it is. If you want to add additional serial numbers, do so by, e.g., adding manufacturerPartInstanceId.

**Comment by eckardg** [comment]
If we agree that only one serial number may be attached to a product, then I agree. If there is the possibility that several serial numbers are attached, then we need this differentiation (and why not wait for the next major change?). There can be the case that a product comes with a serial number from the manufacturer and gets a new label with the customer's serial number at the customer site. In that case it is not possible for the supplier to provide that serial number in the digital twin. And it would be misleading for the customer to expect to find the digital twin.

**Comment by stephanbcbauer** [comment]
@markuskeidl are you working on this PR? Or on a different one? Thx

**Comment by markuskeidl** [comment]
Decision in meeting with @eckardg and @thomas-henn 

- PR is accepted as-is as a clarification of the status quo
- @eckardg and @markuskeidl will create an issue for the Industry Core Committee to re-work the specification and usage of specific asset IDs for manufacturer and customer part IDs and part instance IDs for release 25.06

**Comment by stephanbcbauer** [comment]
THX everybody for clarifying. Will approve and merge the content.

**Pull request merged**